lib/symbols.js

```diff
@@ -31,6 +31,7 @@ const keys = {
   kReplyStartTime: Symbol('fastify.reply.startTime'),
   kReplyErrorHandlerCalled: Symbol('fastify.reply.errorHandlerCalled'),
   kReplyIsRunningOnErrorHook: Symbol('fastify.reply.isRunningOnErrorHook'),
+  kSchemaVisited: Symbol('fastify.kSchemaVisited'),
```

Comment:
I would use "fastify.schemas.visited" since there is the kSchema symbol as sibling

Reply:
got it